src/components/organisms_new/SettingsPane/Settings/Settings.tsx

```diff
@@ -271,10 +269,8 @@ export const Settings = ({
     }
   };
 
-  const doesSchemaExist = useCallback(
-    (k8sVersion: string) => {
-      return existsSync(path.join(String(userDataDir), path.sep, 'schemas', `${k8sVersion}.json`));
-    },
+  const doesSchemaExistCb = useCallback(
```

Comment:
Can we remove this if is not used anymore?

Reply:
resolved